examples/demo_all.pp

```diff
@@ -31,10 +31,12 @@
   include ciscopuppet::demo_command_config
   include ciscopuppet::demo_domain
   include ciscopuppet::demo_interface
+  include ciscopuppet::demo_interface_portchannel
   #include ciscopuppet::demo_interface_service_vni
   include ciscopuppet::demo_ntp
   include ciscopuppet::demo_ospf
   include ciscopuppet::demo_patching
+  include ciscopuppet::demo_portchannel_global
```

Comment:
Does `demo_interface_portchannel` in any way rely on `demo_portchannel_global`?  I would prefer you combine portchannel functionality into one manifiest file and add both `demo_portchannel_global` and `demo_interface_portchannel` into a single `portchannel.pp` and include this in `demo_all.pp`.  This way future portchannel providers can be incrementally added into `portchannel.pp` and they automatically get included in `demo_all.pp`